cmd/podman/ps.go

```diff
@@ -374,22 +374,34 @@ func (p *psTemplateParams) headerMap() map[string]string {
 // getTemplateOutput returns the modified container information
 func getTemplateOutput(containers []*libpod.Container, opts psOptions) ([]psTemplateParams, error) {
 	var psOutput []psTemplateParams
-	var status string
+	var status, ctrID string
```

Comment:
nit: Can you put these into one big var block
```
var (
psOutput []psTemplateParams
status, ctrID string
...
)
```

Reply:
fixed